src/main/java/org/spongepowered/api/event/world/chunk/SaveChunkEvent.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.event.world.chunk;
+
+import com.flowpowered.math.vector.Vector3i;
+import org.spongepowered.api.event.Cancellable;
+import org.spongepowered.api.event.Event;
+import org.spongepowered.api.world.Chunk;
+
+/**
+ * Called when a {@link Chunk} is performing a save.
+ */
+public interface SaveChunkEvent extends Event {
+
+    Vector3i getPosition();
+
+    interface Pre extends SaveChunkEvent, TargetChunkEvent, Cancellable {}
+
+    interface Post extends SaveChunkEvent {}
```

Comment:
Just read Discord, this basically means that the save is going ahead, but may not yet be on the disc. We will want to mention that in the JDs.

Reply:
It was but ultimately that isnt enough for Joni's usecase. He needs to be sure that file wrote to disk. Therefore this is as post as you get, the file is on the disk.